index.html

```diff
@@ -662,6 +662,19 @@ <h2>Algorithms</h2>
 and validate status lists as described by this document.
     </p>
 
+    <p class="issue atrisk"
+       title="Attempting alignment with IETF Token Status List specification">
```

Comment:
"Attempting alignment with IETF OAuth Status List specification"

Reply:
It looks like they're going to change the name to "Token Status List" based on their issue tracker https://github.com/vcstuff/draft-ietf-oauth-status-list/issues/77#issuecomment-1814552128